asyncpg/exceptions/_base.py

```diff
@@ -4,10 +4,11 @@
 # This module is part of asyncpg and is released under
 # the Apache 2.0 License: http://www.apache.org/licenses/LICENSE-2.0
 
-
-import asyncpg
-import sys
-import textwrap
+# PostgresMessageMeta imported for backward compatibility
+from ._postgres_message import (  # noqa: F401
```

Comment:
You can add them to `__all__`. Then you won't need `as` and `noqa`.

Reply:
If I do that, then they will be imported with `from ._base import *` in `asyncpg/exceptions/__init__.py`, and then they will be accessible as `asyncpg.PostgresMessage` and `asyncpg.PostgresMessageMeta`

Reply:
Good point.